File: 4. Java/EX04.java
package Apostila;

import java.util.Scanner;

public class EX04 
{
    public static void main(String[] args) 
    {
        // Declaração das variáveis
        int x, y, som;
        
        // Inicia o Scanner para entrada de dados
        try (Scanner ent = new Scanner(System.in)) 
        {
            // Solicita o valor de X ao usuário
            System.out.print("# Digite o valor de X: ");
            x = ent.nextInt();
            
            // Solicita o valor de Y ao usuário
            System.out.print("# Digite o valor de Y: ");
            y = ent.nextInt();
        }
        
        // Calcula a soma de X e Y
        som = x + y;
        
        // Exibe a soma formatada
        System.out.println("# Soma = " + String.format("%d", som));
    }
}

File: 6. Python/EX04.py
def entDados():
    x = int(input("# Digite o valor de X: "))
    y = int(input("# Digite o valor de Y: "))
    return x, y

def main():
    soma = 0
    a, b = entDados()

    soma = a + b
    print(f"# Soma: {soma}")

if __name__ == '__main__':
    main()